

class CommandAlreadyExecutedException(Exception):
    pass


class CommandNotExecutedException(Exception):
    pass
